docs/pages/create-mod/getting-started.mdx

```diff
@@ -0,0 +1,17 @@
+# Getting started making a Miniapp
+
+A Miniapp is just a JSON file. Part of this JSON is the manifest, defining things like the version and name, and other parts
```

Comment:
This is kind of vague and not the full truth imo

What about something along the lines of

> A Mod consists of a JSON manifest file which conforms to the `ModManifest` type and optionally a backend which can handle requests from the Mod and make external requests. The `ModManifest` type contains metadata about the Mod and describes its functionality and how it should be laid out.

Reply:
will adapt a bit since I think its a bit too technical